examples/response_stream/common/api/reducer_stream/request_body_schema.ts

```diff
@@ -15,5 +15,7 @@ export const reducerStreamRequestBodySchema = schema.object({
   timeout: schema.maybe(schema.number()),
   /** Setting to override headers derived compression */
   compressResponse: schema.maybe(schema.boolean()),
+  /** Boolean lag to enabled/disable 4KB payload flush fix. */
```

Comment:
nit: flag

Reply:
Fixed in [4a8a382](https://github.com/elastic/kibana/pull/165069/commits/4a8a382fb928c5a0a314f9178446000532ffa8ac).